### DISCUSSION TITLE
샘플 예제 prefab 끌어사용했을때 에러

### DISCUSSION THREAD

**Discussion by almondsky** [naverz/zepeto-studio-kor#1064] [5. Q&A]
스크린샷 기능 추가를 하기 위해서 샘플 예제에 프리팹으로 되어있는 zepetoscreenshortmodule 을 제 프로젝트 hierarchy에 끌어넣고, 샘플 예제와 동일하게 script 들을 집어 넣어줬습니다. 문제가,, 똑같이 집어넣었는데,, screen shot mode manager typescript 에서 이 두가지에 빨간줄이 떠서 정상작동을 하지 못하고 있는데,,, 샘플예제와 동일하게 집어넣어 제가 따로 건드린건 없는데,,, 어디가 문제일까요..?
<img width="769" alt="스크린샷 2022-07-25 오후 3 55 57" src="https://user-images.githubusercontent.com/105544091/180716406-d7c5914d-cfb5-40c6-83f5-0373e70e8b66.png">

**Comment by almondsky** [comment]
해결했습니다,!

**Reply by chlguswls32** [reply]
안녕하세요. almondsky님.
저는 아직 스크린샷 기능을 넣어보지않아서 궁금하네요. 어떻게 해결하셧나요?

**Reply by almondsky** [reply]
안녕하세요, 
제페토에서 제공하는 screen shot mode 관련 필요한 prefab 들을 다 끌어오니 해결이 되었습니다,!

**Reply by chlguswls32** [reply]
잘해결되신것같아 다행이네요 ㅎㅎ
